app/src/main/java/com/dcrandroid/activities/TransactionDetailsActivity.java

```diff
@@ -207,51 +211,107 @@ private void loadInOut(ArrayList<TransactionsResponse.TransactionInput> usedInpu
             for (int i = 0; i < outputs.length(); i++) {
                 JSONObject output = outputs.getJSONObject(i);
 
-                if(walletOutputIndices.indexOf(i) != -1){
+                if (walletOutputIndexes.indexOf(output.getInt(Constants.INDEX)) != -1) {
                     continue;
                 }
 
                 JSONArray addresses = output.getJSONArray(Constants.ADDRESSES);
 
-                String address = addresses.length() > 0 ? addresses.getString(0) : getString(R.string.script_bracket);
+                address = addresses.length() > 0 ? addresses.getString(0) : "[script]";
+
+                boolean nullScript = output.getBoolean("NullScript");
 
-                boolean nullScript = output.getBoolean(Constants.NULL_SCRIPT);
```

Comment:
Why remove the `Constants` reference?

Reply:
> Why remove the `Constants` reference?

the problem was in mistake with rebasing master, it is all fixed now